docs/sources/datasources/pyroscope/profilng-tracing-intro.md

```diff
@@ -0,0 +1,19 @@
+---
+title: Profiling and tracing synergies
+menuTitle: Profiling and tracing
+description: Learning about how profiling and tracing work together.
+weight: 250
+aliases:
+  - ../introduction/traces-to-profiles/
+  - ../introduction/profile-tracing/
+keywords:
+  - pyroscope
+  - continuous profiling
+  - tracing
+---
+
+# Profiling and tracing synergies
+
+[//]: # 'Shared content for Trace to profiles in the Pyroscope data source'
+
+{{< docs/shared source="grafana" lookup="datasources/pyroscope-profile-tracing-intro.md" version="<GRAFANA VERSION>" >}}
```

Comment:
Spaces work but underscore separated is the preferred syntax now so that things that split on spaces (like the Hugo link parser) don't get confused.

```suggestion
{{< docs/shared source="grafana" lookup="datasources/pyroscope-profile-tracing-intro.md" version="<GRAFANA_VERSION>" >}}
```